flang/include/flang/Parser/preprocessor.h

```diff
@@ -49,15 +51,21 @@ class Definition {
 
   TokenSequence Apply(const std::vector<TokenSequence> &args, Prescanner &);
 
+  void Print(llvm::raw_ostream &out, llvm::StringRef macroName = "") const;
```

Comment:
`const char *` would be a more portable type for `macroName` for those who use this code outside llvm.

Reply:
Done.